src/plugins/dna_export/src/ExportMSA2MSADialog.ui

```diff
@@ -61,6 +61,51 @@
      </item>
     </layout>
    </item>
+   <item>
+    <widget class="QCheckBox" name="cbIncludeGaps">
+     <property name="text">
+      <string>Include gaps</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QGroupBox" name="gbTranslateAs">
+     <property name="enabled">
+      <bool>false</bool>
+     </property>
+     <property name="title">
+      <string>Translate unknown amino acid as:</string>
```

Comment:
I would not say "translate amino acid" because you translate "nucleic acids" here.
I would rephrase it to "Translate triplets with gaps as:"

Reply:
Done